llap-client/src/java/org/apache/hadoop/hive/registry/impl/ZkRegistryBase.java

```diff
@@ -114,6 +116,7 @@ public abstract class ZkRegistryBase<InstanceType extends ServiceInstance> {
   private PersistentNode znode;
   private String znodePath; // unique identity for this instance
 
+  private String namespace;
```

Comment:
this is supposed to be final as agreed in previous comment

Reply:
fixed